### DISCUSSION TITLE
How do I get the 30GB+ Versions of the IMHEX Program ?

### DISCUSSION THREAD

**Discussion by eddywinch82** [WerWolv/ImHex#1142] [Get Help] [answered]
Hi there,

I have installed the Default Program Version of IMHEX, it is about 7 Gigabytes in Size, from the .msi Installer.

How do I Install and get, the 30 Gigabytes + Versions, I have seen on the internet pictures of Versions with many more features, at a size of of over 30 Gigabytes. Could someone tell me the exact process, of how I should go about, obtaining those versions ? 

Any help would be much appreciated.

Best Wishes From

Eddie Winch

**Comment by WerWolv** [comment, marked as answer]
Hi

I have no idea where you got 7GB and 30GB from, ImHex uses maybe 50MB of storage space in total. The version you can download from the [GitHub release page](https://github.com/WerWolv/ImHex/releases), https://imhex.werwolv.net or https://imhex.download is the full version and the only one that exists. (At least as far as I'm aware of)

**Comment by iTrooz** [comment]
Hey, could you tell us where you saw such information ?

**Reply by eddywinch82** [reply]
Hi guys,

I.e. in the following Website Link :-  

https://github.com/WerWolv/ImHex

In the Pictures you can see there, the size of the IMHEX Program is 31.95 Gigabytes, how do I go about getting that size IMHEX Program with those features ?

**Reply by WerWolv** [reply]
Ohhhh that's the amount of RAM ImHex uses vs the amount the system has available. Those pictures were taken by me on my computer which has 32GB built in :)

**Reply by eddywinch82** [reply]
Hi WerWolv,

I see what you mean, but how do I get the Program running like in those pictures, in the version I have installed, I can't get those features ?

**Reply by WerWolv** [reply]
What features would you like to use? All the ones you see in the picture are available in the version you've downloaded.
I suggest you read through https://docs.werwolv.net to learn more about everything that ImHex can do and how you can make use of it

**Reply by eddywinch82** [reply]
Hi WerWolv,

Sorry about my confusion about the ImHex Program, the file I was opening from a DOS Game i.e. FW190A.PAC didn't have a corresponding .hexpat File for it. so wasn't giving me coloured highlighted text, in the Hex Editor and limited information in the other Options, i.e. in the Pattern Data, Pattern Editor and Data Inspector Windows etc, making me think I had limited options, in the version of ImHex I had installed. Is there a .hexpat File, that can be downloaded for DOS Files or Games etc ?

Eddie :)

**Reply by WerWolv** [reply]
There's the [PE Pattern](https://github.com/WerWolv/ImHex-Patterns/blob/master/patterns/pe.hexpat) that works for modern Windows executables and potentially also for DOS executables, I'm not sure though. Alternatively you can always make your own pattern using the Pattern Editor.
